src/common/UnitTests-CommonLib/UnitTestsCommon.cpp

```diff
@@ -0,0 +1,42 @@
+#include "pch.h"
+#include "common.h"
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace UnitTestsCommon
+{
+    TEST_CLASS(CommonUtils)
+    {
+        std::vector<std::wstring> what{
+            L"TELEGRAM",
+            L"NOTEPAD",
+            L"SUBLIME_TEXT_NOT",
+            L"MICROSOFT VISUAL STUDIO",
+        };
+
+        TEST_METHOD(find_substr_from_vec_test1)
+        {
+            std::wstring where(L"C:\\USERS\\GUEST\\APPDATA\\ROAMING\\TELEGRAM DESKTOP\\TELEGRAM.EXE");
+                bool ans = find_substr_from_vec(where, what);
+                Assert::IsTrue(ans);
+        }
+        TEST_METHOD(find_substr_from_vec_test2)
+        {
+            std::wstring where(L"C:\\WINDOWS\\SYSTEM32\\NOTEPAD.EXE");
+            bool ans = find_substr_from_vec(where, what);
+            Assert::IsTrue(ans);
+        }
+        TEST_METHOD(find_substr_from_vec_test3)
+        {
+            std::wstring where(L"C:\\PROGRAM FILES\\SUBLIME TEXT 3\\SUBLIME_TEXT.EXE");
+            bool ans = find_substr_from_vec(where, what);
+            Assert::IsFalse(ans);
+        }
+        TEST_METHOD(find_substr_from_vec_test4)
+        {
+            std::wstring where(L"C:\\PROGRAM FILES (X86)\\MICROSOFT VISUAL STUDIO\\2019\\COMMUNITY\\COMMON7\\IDE\\DEVENV.EXE");
+            bool ans = find_substr_from_vec(where, what);
+            Assert::IsFalse(ans);
+        }
+    };
```

Comment:
If I had an app with similar name, it will not work.
e.g. this test will fail
```
{
            std::wstring where(L"C:\\USERS\\GUEST\\APPDATA\\ROAMING\\ANOTHER TELEGRAM APP\\TELEGRAM_APP.EXE");
            bool ans = find_substr_from_vec(where, what);
            Assert::IsFalse(ans);
}
```